endpoint/schemas/xregistry_endpoint_protocol_options.json

```diff
@@ -1,5 +1,5 @@
 {
-  "$schema": "http://json-schema.org/draft-07/schema",
+  "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
I'm curious, why is the "#" at the end needed?

Reply:
Apparently that is the common form to refer to draft-07 as I found